fixedint.go

```diff
@@ -9,6 +9,14 @@ import (
 	"strconv"
 )
 
+/*
```

Comment:
Remove

Reply:
note that we are using `v.SetString(strconv.FormatUint(uintVal, 10))` when we convert an int to a string [when weak decoding is enabled]